packages/now-build-utils/test/unit.builds-and-routes-detector.test.js

```diff
@@ -0,0 +1,1161 @@
+const { detectBuilders, detectRoutes } = require('../dist');
```

Comment:
Can this file use typescript now that we have ts jest?

`unit.builds-and-routes-detector.test.ts`

Reply:
👍 converted them, just had to add some casting